omegaconf/omegaconf.py

```diff
@@ -183,7 +184,7 @@ def create(  # noqa F811
     def load(file_: Union[str, IO[bytes]]) -> Union[DictConfig, ListConfig]:
```

Comment:
Please update the typing information on save and load to reflect this change.

Reply:
Done